src/cpp/lpnamer/problem_modifier/LinkParametersCSVOverwriter.cpp

```diff
@@ -0,0 +1,59 @@
+//
+// Created by marechaljas on 16/06/22.
+//
+
+#include "LinkParametersCSVOverwriter.h"
+
+#include "JsonXpansionReader.h"
+#include "LinkdataRecord.h"
+#include "StudyUpdater.h"
+#include "common_lpnamer.h"
+bool LinkParametersCSVOverWriter::open(
+    const std::filesystem::path& linkdataFilename_l) {
+  link_parameters_file_path_ = linkdataFilename_l;
+  inputCsv_l_ = std::ifstream(linkdataFilename_l);
+  tempOutCsvFile_name_ = linkdataFilename_l.string() + "tmp";
+  tempOutCsvFile = std::ofstream(tempOutCsvFile_name_);
+
+  if (!tempOutCsvFile.is_open()) {
+    std::cout << "ERROR: Error writing " + linkdataFilename_l.string() + "."
+              << std::endl;
+    return false;
+  }
+
+  already_warned_ = false;
+  if (!inputCsv_l_.is_open()) {
+    std::cout << "WARNING: Missing file to update antares study : "
+              << linkdataFilename_l << "."
+              << " Unknown valus were populated with 0 in a new created file."
+              << std::endl;
```

Comment:
It would be better yes. Here I don't introduce code, I just moved it. (doesn't mean I didn't add cout elsewhere).
Make a proper usage of a logger averywhere would be a task in itself I think